src/domain/alerts/alerts.repository.interface.ts

```diff
@@ -1,7 +1,10 @@
+import { Contract } from '@/domain/alerts/entities/alerts.entity';
 import { AlertLog } from '@/routes/alerts/entities/alert.dto.entity';
 
 export const IAlertsRepository = Symbol('IAlertsRepository');
 
 export interface IAlertsRepository {
+  addContracts(contracts: Array<Contract>): Promise<void>;
```

Comment:
nitpick: something to consider (can be done at a later stage) – to include some documentation on what is expected by this function.

Reply:
I will address this in a follow-up PR as I have yet to document other areas of recovery.